src/ert/parsing/lark_parser.py

```diff
@@ -250,18 +252,57 @@ def _substitute_args(
     ]
 
 
+class IncludeNode:
+    def __init__(self, parent: "IncludeNode", value: str):
+        self.parent = parent
+        self.value = value
```

Comment:
Does it make sense to call this field `config_file` instead of `value`?

Reply:
Yup, changing